python/src/aiconfig/editor/server/server.py

```diff
@@ -114,17 +117,83 @@ def create() -> FlaskResponse:
     state.aiconfig = AIConfigRuntime.create()  # type: ignore
     return HttpResponseWithAIConfig(message="Created new AIConfig", aiconfig=state.aiconfig).to_flask_format()
 
+@app.route('/api/test_streaming', methods=["POST"])
```

Comment:
You may as well make the real endpoint. It's just us using it anyway; we're not committing to anything big here.